codecov_cli/services/report/__init__.py

```diff
@@ -118,7 +116,7 @@ def send_reports_result_get_request(
     enterprise_url,
     fail_on_error=False,
 ):
-    headers = get_token_header_or_fail(token)
+    headers = get_token_header(token)
```

Comment:
`get_token_header` currently doesn't take an optional argument ([according to the type annotation](https://github.com/codecov/codecov-cli/blob/6efe50c4d4c4b51a9c7799e48c1042288ba64378/codecov_cli/helpers/request.py#L113)). Care to fix that as well? :p

Reply:
certainly haha